src/stream/src/from_proto/stream_cdc_scan.rs

```diff
@@ -98,6 +98,8 @@ impl ExecutorBuilder for StreamCdcScanExecutorBuilder {
             state_table,
             node.rate_limit,
             disable_backfill,
+            1,
```

Comment:
`snapshot_interval` will default to `1`, and we allow user to config different value in WITH clause https://github.com/risingwavelabs/risingwave/pull/16426